### PR TITLE
Release 1.1.1

### PR DESCRIPTION
**Pull request by SGenheden**
…mp version

### CODE REVIEW

**Comment by SGenheden** [comment]
Change project name. Update RDChiral dependency. Update changelog. Bump version

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ If you find an issue on a Windows or macOS platform, we might be able to fix, bu
 
 Setup your python environment and then run
 
-    pip install rxnutils
+    pip install reaction-utils
```
Comment (CKannas) [inline README.md:28]: Should we also change the title to `# reaction-utils`?

File: docs/conf.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 project = "ReactionUtils"
```
Comment (CKannas) [inline docs/conf.py:6]: Should we change this to `project = "reaction-utils"`?

File: docs/index.rst [hunk 3]
```diff
@@ -21,7 +21,7 @@ For most users it is as simple as
 
 .. code-block::
 
-    pip install rxnutils
+    pip install reaction-utils
```
Comment (CKannas) [inline docs/index.rst:24]: Should we change the title to `reaction-utils documentation`?

**Pull request merged**